pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest.java

```diff
@@ -2194,7 +2194,7 @@ public void testPersistentTopicsExpireMessages() throws Exception {
         Thread.sleep(1000);
         admin.topics().expireMessages("persistent://prop-xyz/ns1/ds2", "my-sub1", 1);
         // Wait at most 2 seconds for sub1's message to expire.
```

Comment:
It's probably worth updating this comment, as the wait is now the default setting instead of the specified 2 seconds.